### DISCUSSION TITLE
I cannot create a new repo

### DISCUSSION THREAD

**Discussion by DoaaMosalam** [community/community#78727] [Repositories]
### Select Topic Area

General

### Body

When creating a new repository repo I get this error message “Owner is being deleted,so new repository cannot be created” Signs that when I modify a project on github and upload it, it rises and unpdate. 
![IMG_0861](https://github.com/community/community/assets/40686512/19f6ee4b-ae9f-4691-bd4c-42218119ca04) can anyone fixed this, please.

**Comment by DoaaMosalam** [comment]
I cannot created new repository from browser and mobile

**Comment by elijahbc93** [comment]
I get the same issue. What's going on

**Comment by theapplegates** [comment]
Just jumping in. Same here. Ticket #2493218
